experimental/veggieseasons/lib/screens/settings.dart

```diff
@@ -211,6 +213,44 @@ class SettingsScreen extends StatelessWidget {
     );
   }
 
+  SettingsItem _buildRestoreDefaultsItem(
+      BuildContext context, Preferences prefs) {
+    return SettingsItem(
+      label: 'Restore Defaults',
+      icon: SettingsIcon(
+        backgroundColor: Styles.iconRed,
+        icon: Styles.resetIcon,
+      ),
+      content: SettingsNavigationIndicator(),
+      onPress: () {
+        showCupertinoDialog<void>(
+          context: context,
+          builder: (context) => CupertinoAlertDialog(
+            title: Text('Are you sure?'),
+            content: Text(
+              'Are you sure you want to reset the current preferences?',
```

Comment:
nit: Rather than "preferences," how about "settings," since that's the word used in the title for this screen?

Reply:
sounds good